src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -814,6 +815,13 @@ def load_arguments(self, _):
                 'For more information on "Auto" mode see aka.ms/aks/nap.'
             )
         )
+        # in creation scenario, use "localuser" as default
+        c.argument(
+            'ssh_access',
+            arg_type=get_enum_type(["disabled", "localuser"]),
```

Comment:
define a list of available options and refer to the list?
`ssh_accesses = [CONST_SSH_ACCESS_DISABLED, CONST_SSH_ACCESS_LOCALUSER]`
`arg_type=get_enum_type(ssh_accesses)`